docs/QUICKSTART.md

```diff
@@ -135,76 +158,122 @@ This tutorial demonstrates the functionality of Eraser and validates that non-ru
+    docker.io/library/hello-world:latest                                                                  application/vnd.docker.distribution.manifest.list.v2+json sha256:80f31da1ac7b312ba29d65080fddf797dd76acfb870e677f390d5acba9741b17 6.9 KiB   linux/386,linux/amd64,linux/arm/v5,linux/arm/v7,linux/arm64/v8,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x,windows/amd64 io.cri-containerd.image=managed
+    docker.io/library/hello-world@sha256:80f31da1ac7b312ba29d65080fddf797dd76acfb870e677f390d5acba9741b17 application/vnd.docker.distribution.manifest.list.v2+json sha256:80f31da1ac7b312ba29d65080fddf797dd76acfb870e677f390d5acba9741b17 6.9 KiB   linux/386,linux/amd64,linux/arm/v5,linux/arm/v7,linux/arm64/v8,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x,windows/amd64 io.cri-containerd.image=managed
     ```
 
-    Pods will run to completion and the images will be removed.
-
-    ```bash
-    $ kubectl get pods -n eraser-system
-    eraser-system        eraser-controller-manager-6d6d5594d4-phl2q   1/1     Running     0          4m16s
-    eraser-system        eraser-kind-control-plane                    0/1     Completed   0          22s
-    eraser-system        eraser-kind-worker                           0/1     Completed   0          22s
-    eraser-system        eraser-kind-worker2                          0/1     Completed   0          22s
-    ```
-
-    The `imagelist` custom resource status field will contain the status of the last job. The success and failure counts indicate the number of nodes the Eraser agent was run on.
-
-    ```bash
-    $ kubectl describe ImageList imagelist
-    ...
-    Status:
-      Failed:     0
-      Success:    3 
-      Timestamp:  2022-02-25T23:41:55Z
-    ...
-    ```
-
-    By default, successful jobs will be deleted after a period of time. You can change this behavior by setting the following flags in the eraser-controller-manager:
-
-    - `--job-cleanup-on-success-delay`: Seconds to delay job deletion after successful runs. 0 means no delay. Defaults to `0`.
-    - `--job-cleanup-on-error-delay`: Seconds to delay job deletion after errored runs. 0 means no delay. Defaults to `86400` (24 hours).
-    - `--job-success-ratio`: Ratio of successful/total runs to consider a job successful. 1.0 means all runs must succeed. Defaults to `1.0`.  
-
-6. Verify the unused images are removed
-
-    ``` bash
-    docker exec kind-worker ctr -n k8s.io images list | grep hello-world
-    ```
-
-    If the image has been successfully removed, there will be no output. 
-
-7. Tear down your cluster
-
-    ```bash
-    kind delete cluster
-    ```
\ No newline at end of file
+## Automatically cleaning images
+
+After deploying Eraser, it will automatically clean images in a regular interval. This interval can be set by `--repeat-period` argument to `eraser-controller-manager`. The default interval is 24 hours.
```

Comment:
The repeat period needs to be better documented in general. For job-cleanup delays we are require seconds, but for repeat period we are using input to `ParseDuration`. This should probably be fixed in the code. 
```
Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```